config/default.json

```diff
@@ -0,0 +1,19 @@
+{
+    "contracts": {
+        "relayHubAddress": "0xDA7Ce79725418F4F6E13Bf5F520C89Cec5f6A974",
+        "relayVerifierAddress": "0x03F23ae1917722d5A27a2Ea0Bcc98725a2a2a49a",
+        "deployVerifierAddress": "0x73ec81da0C72DD112e06c09A6ec03B5544d26F05"
+    },
+    "app": {
+        "url": "http://localhost:8090",
+        "port": 8090,
+        "devMode": true,
+        "customReplenish": false,
+        "logLevel": 1,
+        "workdir": "/Users/borja/iovlabs/projects/temp/PP-358/environment"
+    },
+    "blockchain": {
+        "gasPriceFactor": 1,
+        "rskNodeUrl": "http://127.0.0.1:4444"
+    }
+}
```

Comment:
put this in `./config/local.json(5)` please

Reply:
I'm still waiting for your PR to me merged to do that :)